.circleci/config.yml

```diff
@@ -106,6 +106,45 @@ jobs:
     steps:
       - build-steps
 
+  build-macos:
+      macos:
+        xcode: 11.6.0
+      steps:
+       - checkout-all
+       - run:
+          name: get conda
+          command: |
+            brew install wget
+            wget https://repo.anaconda.com/archive/Anaconda3-2021.11-MacOSX-x86_64.sh -O ~/anaconda.sh
+            bash ~/anaconda.sh -b -p $HOME/anaconda
+            source ~/anaconda/bin/activate
+            conda init zsh
+            source ~/.zshrc
+            conda env create -f ./conda/macos.yml
```

Comment:
PLEASE PLEASE PLEASE
Don't do it like that. Let's try to avoid putting real code into config.yml.
There's `getconda` in readies now - let's make it work through there.
I would also like very much to avoid zsh in automation - but let's take it offline, I'll lay out my arguments.